Symfony/CS/Fixer/Contrib/PhpUnitNoDedicateAssertFixer.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Contrib;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class PhpUnitNoDedicateAssertFixer extends AbstractFixer
+{
+    private $fixMap = array(
+        'is_nan' => array(false, 'assertNan'),
+        'is_null' => array('assertNotNull', 'assertNull'),
+        'file_exists' => array('assertFileNotExists', 'assertFileExists'),
+        'empty' => array('assertNotEmpty', 'assertEmpty'),
+    );
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+        $searchSequence = $this->getSearchSequence();
+
+        $index = 1;
+        while (null !== $candidate = $tokens->findSequence($searchSequence, $index)) {
```

Comment:
:-1: In my opinion the assignment inside `while()` was more readable. 
Now we have the `findSequence` call twice.